packages/ontario-frontend-cli/commands/ontario-create-app/handleCreateApp.js

```diff
@@ -115,47 +105,78 @@ async function generateProjectFiles(newProjectPath, conf) {
       logger.error(
         `Failed to render and write ${outputPath}: ${error.message}`,
       );
-      // Throw the error to halt the execution if one file fails to generate
       throw error;
```

Comment:
Will this throw the logger.error message above? If not can we specify a user friendly error message here?

Reply:
I think this is just propagating the logger.error to the calling function but let me know if I understood that correctly.

Reply:
It's because of the try/catch chaining that is clogging the code. I went into a huge deep dive refactoring, but broke it out into the branch `matt/refactor-utils-trycatch` to avoid the big scope creep